main.go

```diff
@@ -81,17 +93,20 @@ func main() {
 	var nsDeleteWorkers int
 	var nodeDeleteWorkers int
 	var nodeLabelSyncWorkers int
+	var nodeFencerWorkers int
 	var pvcLabelSyncWorkers int
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&apiSecretPath, "api-secret-path", "/etc/storageos/secrets/api", "Path where the StorageOS api secret is mounted.  The secret must have \"username\" and \"password\" set.")
 	flag.StringVar(&apiEndpoint, "api-endpoint", "storageos", "The StorageOS api endpoint address.")
-	flag.DurationVar(&apiPollInterval, "api-poll-interval", 5*time.Second, "Frequency of StorageOS api polling.")
+	flag.DurationVar(&volumePollInterval, "volume-poll-interval", 5*time.Second, "Frequency of StorageOS volume polling.")
+	flag.DurationVar(&nodePollInterval, "node-poll-interval", 5*time.Second, "Frequency of StorageOS node polling.")
+	flag.DurationVar(&volumeExpiryInterval, "volume-expiry-interval", time.Minute, "Frequency of cached StorageOS volume re-validation.")
+	flag.DurationVar(&nodeExpiryInterval, "node-expiry-interval", 15*time.Minute, "Frequency of cached StorageOS node re-validation.")
```

Comment:
The cache is set to expire at 15 minutes. Since we don't depend on cache flush for anything, we can increase the default even more if we want to.

Reply:
Let's set to 1 hour